pyanaconda/ui/cockpit/anaconda-webui/Makefile.am

```diff
@@ -38,6 +38,13 @@ dist_noinst_DATA = \
 $(WEBPACK_TEST): $(LIB_TEST) $(shell find src/ -type f) $(NODE_MODULES_TEST) package.json webpack.config.js
 	NODE_ENV=production $(CURDIR)/node_modules/.bin/webpack
 
+watch:
+	rm -f dist/*
+	NODE_ENV=$(NODE_ENV) node_modules/.bin/webpack --watch
+
+rsync:
+	RSYNC=test-updates make watch
```

Comment:
There should be a way how to avoid dependency on the system configuration. The `test-updates` in the ssh config could be a no-go for someone and also it's not really usable for multiple VMs running.

Reply:
I plan on adding another way to do this, but this needs to be implemented upstream in cockpit's repo as currently this rsync plugin does not support custom SSH port, which our test VM has (see 22000)

Setting the ssh host for development ease is something we advertise in cockpit as well, https://github.com/cockpit-project/cockpit/tree/main/test/#helpful-tips

I agree it can be improved, but as this needs changes to the upstream cockpit project, I suggest not blocking, as I want people to be able to iterate in a fast way for the workshop on thursday.

The other way would be for everyone to manually scp all the time, which is rather annoying.

Reply:
@jkonecny12 - for the record https://github.com/cockpit-project/cockpit/issues/16817